draft-ietf-quic-transport.md

```diff
@@ -2759,10 +2759,16 @@ ordered byte-stream.  Data received out of order MUST be buffered for later
 delivery, as long as it is not in violation of the receiver's flow control
 limits.
 
-An endpoint MUST NOT send data on any stream without ensuring that it is within
-the data limits set by its peer.  The cryptographic handshake stream, Stream 0,
-is exempt from the connection-level data limits established by MAX_DATA.  Stream
-0 is still subject to stream-level data limits and MAX_STREAM_DATA.
+An endpoint MUST NOT send data on any stream without ensuring that it
+is within the data limits set by its peer.  The cryptographic
+handshake stream, Stream 0, is exempt from the connection-level data
+limits established by MAX_DATA. Data on stream 0 other than the
+TLS ClientHello is still subject to stream-level data limits and
+MAX_STREAM_DATA. The TLS ClientHello is exempt from flow control because it needs
+to be sent in one piece regardless of the server's flow control
+state. This rule applies even for 0-RTT handshakes where the
+remembered value of MAX_STREAM_DATA would not permit sending
```

Comment:
nit: "would not permit" --> "does not permit"

Reply:
I made the change, but I actually think this is wrong because this is a counterfactual. I.e., it wouldn't permit it if taken literally, but we do not.

Reply:
Ah, right, good point. Your wording seems better, so go with it.